### DISCUSSION TITLE
Does esm.sh read package.json exports or module fields?

### DISCUSSION THREAD

**Discussion by jedwards1211** [esm-dev/esm.sh#685] [Q&A]
Skimming the code base it looks like it may only look at the CJS and try to convert it to ESM, even in dual packages that provide both CJS and ESM with `main`, `module`, and `exports` in the `package.json`.

**Comment by ije** [comment]
hey @jedwards1211 can you please post the links? and yes esm.sh reads the `exports` field of package.json
